components/brave_ads/browser/analytics/p3a/notification.cc

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/browser/analytics/p3a/notification.h"
+#include "base/metrics/histogram_macros.h"
+#include "brave/components/brave_ads/core/public/prefs/pref_names.h"
+#include "components/prefs/pref_service.h"
+
+namespace brave_ads {
+
+constexpr char kNotificationPositionHistogramName[] =
+    "Brave.Rewards.CustomNotificationPosition";
+constexpr base::TimeDelta kRecordPositionDebounceDelay = base::Seconds(5);
```

Comment:
if definition is not used externally, it is better to move it inside unnamed namespace

Reply:
removed, since we no longer need debounce logic